### PR TITLE
Python11 windows automation upgrade

### PR DESCRIPTION
**Pull request by nadimurrashid**
## PR Type

PR_TYPE

## PR Checklist

- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] A changelog entry has been made.
- [ ] Version number has been updated.
- [ ] Required modules have been added to respective "requirements*.txt" files.
- [ ] Relevant Test Cases added to this description (below).
- [ ] (Team) Label with affected action categories and semver status.

## Test Cases
- [TEST-0000](https://zeuz.zeuz.ai/Home/ManageTestCases/Edit/TEST-0000/)

### CODE REVIEW

File: Apps/aiplugin/background.js [hunk 1]
```diff
@@ -3,8 +3,8 @@ const tabs = {};
 const inspectFile = 'inspect.js';
 const activeIcon = 'active-64.png';
 const defaultIcon = 'default-64.png';
-let zeuz_url = '__ZeuZ__UrL_maPP'
-let zeuz_key = '__ZeuZ__KeY_maPP'
+let zeuz_url = 'https://apollo.zeuz.ai'
```
Comment (sazid) [inline Apps/aiplugin/background.js]: @nadimurrashid remove these

File: Apps/aiplugin/background.js [hunk 2]
```diff
@@ -3,8 +3,8 @@ const tabs = {};
...
+let zeuz_key = 'a6fafbd1-b15f-4ece-a13f-3c3ddfd4265f'
```
Comment (sazid) [inline Apps/aiplugin/background.js]: @nadimurrashid make sure to regenerate a new api key.

Comment (sazid) [inline Apps/aiplugin/background.js]: @Muntasib-creator this will need to be put into gitignore.

Comment (Muntasib-creator) [reply Apps/aiplugin/background.js]: @sazid it is in the ignore

**Pull request closed without merging**